lib/engine/game/base.rb

```diff
@@ -546,8 +552,12 @@ def inspect
         "#{self.class.name} - #{self.class.title} #{players.map(&:name)}"
       end
 
-      def result
+      def result_players
```

Comment:
why is this changed

Reply:
It's a new method, just git showing the diff weird. the original `result` method is right below.

This is the ability to choose while players are shown in the end game scoring. Union bank is represented here as a player but it shouldn't have it's own score at the end.